package.json

```diff
@@ -44,6 +44,7 @@
     "babel-core": "^5.8.25",
     "babel-loader": "^5.3.2",
     "chai": "^3.3.0",
+    "core-js": "^2.4.1",
```

Comment:
Interesting! This is specifically for the few tests which are using `Map`, not for the actual library, so I'm not super worried about spec compliance here. I can switch it out, though.